Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -282,11 +282,13 @@ public class CRTClientEngine: HTTPClient {
                                         }
 
                                         var hasMoreChunks = true
+                                        var currentChunkBodyIsEmpty = false
                                         while hasMoreChunks {
                                             // Process the first chunk and determine if there are more to send
                                             hasMoreChunks = try await chunkedStream.chunkedReader.processNextChunk()
+                                            currentChunkBodyIsEmpty = chunkedStream.chunkedReader.getCurrentChunkBody().isEmpty
```

Comment:
swiftlint warning! -- we should change these to failures rather than warns since we don't ever approve PRs that have this, will take this up

Reply:
The bigger issue here is that this is not a long line of code.  It's so far nested that much of the line length is just leading space.  It suggests this method should be broken into submethods

Reply:
Addressed warning.